frontend/apps/docs/content/docs/parser/supported-formats/rails.mdx

```diff
@@ -2,4 +2,115 @@
 title: schema.rb (Ruby on Rails)
 ---
 
-TBD
+If you’re using Ruby on Rails with ActiveRecord, in most cases you can automatically generate a useful ER diagram with Liam ERD. This page provides instructions and tips for generating an ER diagram in a Rails project.
+
+## ActiveRecord and db/schema.rb
+
+When a Rails application manages migrations using ActiveRecord, the latest database structure is typically described in `db/schema.rb`.  
+Since `db/schema.rb` is recommended to be kept under version control, most projects will have it in their Git repository.
+
+When using Liam CLI, specify `--format schemarb` and `--input db/schema.rb` as follows:
+
+<Tabs items={["npm", "pnpm", "yarn", "bun"]}>
+  <Tab value="npm" copyable>npx @liam-hq/cli erd build --format schemarb --input db/schema.rb</Tab>
+  <Tab value="pnpm" copyable>pnpm dlx @liam-hq/cli erd build --format schemarb --input db/schema.rb</Tab>
+  <Tab value="yarn" copyable>yarn dlx @liam-hq/cli erd build --format schemarb --input db/schema.rb</Tab>
+  <Tab value="bun" copyable>bun x @liam-hq/cli erd build --format schemarb --input db/schema.rb</Tab>
+</Tabs>
```

Comment:
no syntax color. but it seems to be ok. https://github.com/liam-hq/liam/pull/539#issuecomment-2609102959